static/_redirects

```diff
@@ -101,10 +101,9 @@
 /guides/artifacts/api /ref/python/public-api/api#artifact 301
 /guides/artifacts/artifacts-core-concepts /guides/artifacts 301
 /guides/artifacts/artifacts-faqs /support/index_artifacts 301
-/guides/artifacts/project-scoped-automations /guides/core/automations 301
 /guides/artifacts/dataset-versioning /guides/artifacts 301
 /guides/artifacts/model-versioning /guides/core/registry/ 301
-/guides/artifacts/project-scoped-automations /guides/core/automations/project-scoped-automations 301
+/guides/core/artifacts/project-scoped-automations /guides/core/automations/automation-events/#project 301
```

Comment:
@mdlinville This redirect works...IF there is a trailing slash. i.e. /guides/core/artifacts/project-scoped-automations/ 

Does this mean that they all need a trailing slash?

Reply:
Hmm...I guess not. I tried your PR and it works. 🤔